src/datasources/asset/types/AssetSummaryQuery.types.ts

```diff
@@ -1,4 +1,9 @@
 import { DataQuery } from '@grafana/schema'
 
 export interface AssetSummaryQuery extends DataQuery {
-}
+    total: number;
+    active: number;
+    notActive: number;
+    approachingRecommendedDueDate: number;
+    pastRecommendedDueDate: number;
+  }
```

Comment:
This doesn't make sense—the `DataQuery` type is meant to represent the query a user builds for a data source. This type is more like an `AssetSummaryResponse`.